src/openms/source/METADATA/PeptideIdentification.cpp

```diff
@@ -269,6 +271,73 @@ namespace OpenMS
     }
     return filtered;
   }
+
+  void PeptideIdentification::fillConsensusPepIDMap_help(vector<PeptideIdentification>& cpep_ids,
```

Comment:
make this a lambda
```
auto lamda = []((const vector<PeptideIdentification>& cpep_ids, const map<String, StringList>& identifier_to_msrunpath, multimap<String, PeptideIdentification*>& customID_to_cpepID) {
      for (PeptideIdentification &cpep_id : cpep_ids)
  {
      auto x =  build_uid_from_pep_id()...
      customID_to_cpepID.insert(make_pair(UID, &cpep_id));
};

```
}
                              multimap<String, PeptideIdentification*>& customID_to_cpepID)